Validation/CaloTowers/interface/CaloTowersValidation.h

```diff
@@ -34,9 +34,9 @@ class CaloTowersValidation : public DQMEDAnalyzer {
  public:
    CaloTowersValidation(edm::ParameterSet const& conf);
   ~CaloTowersValidation();
-  virtual void analyze(edm::Event const& e, edm::EventSetup const& c);
-  virtual void beginRun() ;
-  virtual void endRun() ;
+  virtual void analyze(edm::Event const& e, edm::EventSetup const& c) override;
+  virtual void beginRun();
```

Comment:
I agree that `analyzer` needs an override. What I'm saying is `beginRun` and `endRun` are never going to be called.